src/pages/enterprise.js

```diff
@@ -84,7 +84,7 @@ const Enterprise = () => (
                   Offering Pixie support?
                 </p>
                 <a
-                  href='https://github.com/pixie-labs/pixie/pulls'
+                  href='https://github.com/pixie-io/px.dev/pulls'
```

Comment:
this is changing the repo too

Reply:
This was intentional - I was fixing what I think is a bug. If you want to change this website, you'd need the px.dev repo.